docs/docs/howto/installation/run-source.rst

```diff
@@ -1,34 +1,32 @@
 Run Klaw from source
 ====================
 
-Klaw is a Java project, you will need `Maven <https://maven.apache.org/>`_ installed to run this project from source.
+Klaw is a Java project, and you need `Maven <https://maven.apache.org/>`_ installed to run this project from the source.
```

Comment:
This is not the case anymore, right? As we have `mvnw`, this will download maven for you

Reply:
Agree, updated.